libs/sdk-ui-pivot/src/impl/data/loadingRenderer.tsx

```diff
@@ -0,0 +1,40 @@
+// (C) 2019-2020 GoodData Corporation
+import { ICellRendererParams } from "@ag-grid-community/all-modules";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+import React from "react";
+import { VALUE_CLASS } from "../base/constants";
+import { TableFacade } from "../tableFacade";
+import { ICorePivotTableProps } from "../../publicTypes";
+
+/**
+ * Creates a loading renderer functional component which the table uses for rendering cells that will also show loading
+ * indicators as data for them is being loaded.
+ *
+ * NOTE: keep in mind that this loading renderer IS NOT used to configure ag-grids built-in loading renderer. This
+ * renderer is essentially a custom cell renderer which our table impl uses for all cells of the left-most table column.
+ *
+ * @param _table
+ * @param _props
+ */
+export function createLoadingRenderer(
+    _table: TableFacade,
```

Comment:
Why do you need the factory? You are not "baking in" anything, the parameters of the factory are not used.

Reply:
this is here for consistency with other cell renderers which do need to access table / table descriptor. the loading renderer should be just a decorator that adds the indicator on top the normal cell renderer.

one refactor ahead, if we want to use renderer composition (loading indicator on top of the standard cell renderer) we will need these parameters.

Reply:
Fair, leave it in :)